android/java/org/chromium/chrome/browser/ntp/BraveNewTabPageLayout.java

```diff
@@ -372,7 +372,8 @@ public void run() {
 
     private void setNtpRecyclerView(LinearLayoutManager linearLayoutManager) {
         mIsTopSitesEnabled = NtpUtil.shouldDisplayTopSites();
-        mIsBraveStatsEnabled = NtpUtil.shouldDisplayBraveStats();
```

Comment:
return false from inside this function instead. we wouldn't have to make changes at multiple places then